package.json

```diff
@@ -21,11 +21,13 @@
     "body-parser": "^1.18.3",
     "compression": "^1.7.4",
     "cookie-session": "^2.0.0-beta.3",
+    "cron": "^1.8.2",
```

Comment:
Could you revert the merging of the wiping PR and rebase on master please? Now that you've shown it's not required to wipe guild data, I'd like to be able to merge this separately.